fitwsarah_frontend/src/views/AboutMe/AboutMe.js

```diff
@@ -81,16 +111,25 @@ function AboutMe() {
                         </div>
                     </div>
                 </div>
-
-                <div className="container1">
-                    <div className="box">
-
-                    </div>
+                {isAuthenticated && (
+                <div className="container1">  
                     <div className="box">
-                        {/* for the feedback */}
+                    <h2 className="recentTitle">{t('feedbacks')}<br /></h2>
+                    <Swiper slidesPerView={2}spaceBetween={15}className="mySwiper">
+                        <div class="scrolling-wrapper-flexbox">
+                            {feedbacks.map(feedback => (
+                                <SwiperSlide>
```

Comment:
could you tell me what Swiper does

Reply:
Sure, it simply creates a swipeable container wrapping all the vsisible feedback.